### ISSUE TITLE
RightCurly: Curly brace '}' should be on line by itself not reported for method with annotation

### ISSUE THREAD

**Issue by dhendriks** [checkstyle/checkstyle#1014]
For the RightCurly check, to check that '}' should be on a line by itself, at the start of that line, I use:

```
    <module name="RightCurly">
      <property name="option" value="alone"/>
      <property name="tokens" value="CLASS_DEF,METHOD_DEF,CTOR_DEF,LITERAL_FOR,LITERAL_WHILE,LITERAL_DO,STATIC_INIT,INSTANCE_INIT"/>
    </module>
```

The following code however, where the '}' is not on a line by itself, does not give any warnings:

```java
    @Override
    public String getTypeText() { return "int"; }
```

The following code however, which is the same except the '@Override' annotation is removed, does give a warning:

```java
    public String getTypeText() { return "int"; }
```

This seems like a bug to me?

**Comment by romani** [comment]
merged
